middlewares/validators/user.js

```diff
@@ -213,11 +213,27 @@ const validateImageVerificationQuery = async (req, res, next) => {
   }
 };
 
+async function validateUpdateRoles(req, res, next) {
+  const schema = joi.object().strict().min(1).max(1).keys({
+    member: joi.boolean(),
+    archived: joi.boolean(),
+  });
+
+  try {
+    await schema.validateAsync(req.body);
+    next();
+  } catch (error) {
+    logger.error(`Error validating updateRoles query params : ${error}`);
+    res.boom.badRequest(error.details[0].message);
```

Comment:
what's your opinion on sending a custom error message rather than the exact error message?
The benefits of it would be: 
1. Easier testing as we know the response
2. We should not return the exact error as it might leak some critical data about our backend

Reply:
```
catch (error) {
    logger.error(`Error validating updateRoles query params : ${error}`);
    res.boom.badRequest(error.details[0].message);
    
```
    You are write
    so I follow consistent way like above others code 
    `res.boom.badRequest(error.message);`
    
    or 
     `res.boom.badRequest('invalid role`);`

Reply:
Done after discussion with ritik